README.md

```diff
@@ -54,10 +62,19 @@ In all projects there needs to be a reliable method of source/version control. [
 ### Salesforce Specific Source Control
 Salesforce often uses Production as the source of truth, which can be problematic. In which case see the `Source Code Management` section of [Software Development in Salesforce](https://www.linkedin.com/pulse/software-development-salesforce-tuan-lu/).
 
-## Git Repo Hosting
+## Source Control Tools
+Source Code Management is typically required for larger heavily customized implementations of Salesforce. Its benefits are as follows. 
+Maintains historical record of changes for code review, rollback of changes and management of packages to be released. 
+Facilitates means for continuous integration across packages and sandboxes.
+Facilitates release management by using Metadata Packages.  
+Salesforce customizations can be accessed as text or XML through a number of interfaces. The most common are the Metadata API and the SFDX commandline tool which uses the Metadata API.
+
 One of the following should be used:
 - GitHub
 - BitBucket
+- Subversion
```

Comment:
Do we use subversion, VSTS, or TFS, on any *non-legacy* projects? If so why not git? Is this something customers would request?

Reply:
We have used subversion, VSTS, and TFS.  VSTS and TFS might come up more and more with VS Code as that is what a lot of .Net use for Source Control.  I don't think it hurts to leave them there as we are just showing examples of what clients may use.  Maybe we could add a *preferred next to our preferred Source Control, IDE, etc.?